deployment/environment_full_linux.yml

```diff
@@ -134,7 +134,7 @@ dependencies:
   - libedit=3.1.20170329=hf8c457e_1001
   - libffi=3.2.1=he1b5a44_1006
   - libgcc-ng=8.2.0=hdf63c60_1
-  - libgfortran-ng=7.3.0=hdf63c60_0
+  - libgfortran-ng=7.2.0=hdf63c60_3
   - libglu=9.0.0=hf484d3e_1000
   - libiconv=1.15=h516909a_1005
   - liblapack=3.8.0=8_mkl
```

Comment:
You need to change this to `liblapack=3.8.0=8_openblas`

Reply:
I think we tried to get rid of OpenBLAS here and use mkl, but it didn't work because Fenics seemed to **need** OpenBLAS somehow.

Reply:
I wrote down what the problem was [here](https://github.com/conda-forge/fenics-feedstock/issues/82#issuecomment-496645709).